### DISCUSSION TITLE
港澳台番剧页面脚本无法使用

### DISCUSSION THREAD

**Discussion by Neo-Ze** [the1812/Bilibili-Evolved#3254] [提问 / Q&A]
不知道是不是因为使用了“解除B站区域限制 v8.2.16”这个脚本，Evolved失效，设置弹出来什么都没有😥
![0](https://user-images.githubusercontent.com/81399833/163705519-ebb59ac8-5e57-4790-9d9b-477ccc6a9ce7.gif)

**Comment by the1812** [comment]
不支持港澳台

**Comment by Tinhone** [comment]
Bilibill Evolved v1.12.22 在安装脚本“解除B站区域限制 v8.2.16”后，港澳台番剧界面和普通界面均可正常使用Bilibill Evolved的功能
应该是v2版本专门对这类页面作了限制

<del>幸好没更新（暗喜）</del>

**Reply by the1812** [reply]
也没专门搞限制, 只是不保证港澳台的体验. 之前也有报这个问题, 不过我装了解除区域限制后也复现不了就懒得管了

**Reply by Kal-tsit63** [reply]
真的没法复现吗，我这边也是坏的。不光坏了，搜索框也会变得特别奇怪。。。

**Reply by MY1L** [reply]
我也遇到了，而且点击设置按钮一下子页面都爆了（指页面排版炸裂）
搜索[Issues?q=解除B站区域限制](https://github.com/the1812/Bilibili-Evolved/issues?q=%E8%A7%A3%E9%99%A4B%E7%AB%99%E5%8C%BA%E5%9F%9F%E9%99%90%E5%88%B6)发现许多人有类似问题……
